docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -190,6 +180,36 @@ match the hex value for `port`. The remote address depends on which node initiat
     }
     ```
 
+### admin_removePeer
+
+Removes a [static node](../Configuring-Pantheon/Networking/Managing-Peers.md#static-nodes).  
+
+**Parameters**
+
+`string` : [Enode URL](../Configuring-Pantheon/Node-Keys.md#enode-url) of peer to remove
+
+**Returns**
+
+`result` : `boolean` - `true` if peer added or `false` if peer not a [static node]((../Configuring-Pantheon/Networking/Managing-Peers.md#static-nodes)). 
```

Comment:
Should be removed instead of added I guess.
```suggestion
`result` : `boolean` - `true` if peer removed or `false` if peer not a [static node]((../Configuring-Pantheon/Networking/Managing-Peers.md#static-nodes)). 
```